packages/esm-patient-medications-app/src/order-basket/medication-order-form.component.tsx

```diff
@@ -54,19 +128,23 @@ export default function MedicationOrderForm({
           <>
             <span>
               <strong className={styles.dosageInfo}>
-                {capitalize(orderBasketItem.commonMedicationName)} ({orderBasketItem.dosage.dosage})
+                {capitalize(orderBasketItem.commonMedicationName)} {doseWithUnitsLabel && `(${doseWithUnitsLabel})`}
               </strong>{' '}
-              <span className={styles.bodyShort01}>
-                &mdash; {orderBasketItem.route.name} &mdash; {orderBasketItem.dosageUnit.name} &mdash;{' '}
-              </span>
-              <span className={styles.caption01}>{t('dose', 'Dose').toUpperCase()}</span>{' '}
-              <strong className={styles.dosageInfo}>{orderBasketItem.dosage.dosage}</strong>
+              {template && (
```

Comment:
Nit: `template` is not a boolean, so there might be occasions where this check leads to a `0` getting rendered. A ternary operator would work better in this circumstance.

```tsx
  {template ? () : null}
```

Reply:
So template is type "object", it's very unlikely for an order template's datatype to be otherwise. It's either an object if in a given scenario on was found or null.